### PR TITLE
Maintainer doc updates

### PR DESCRIPTION
**Pull request by chawes13**
**Adds**:
- Code of Conduct
- Contribution guide
- Pull Request Template
- Issue Templates (Bug, Feature)
- New section in README referencing Contribution guide

**Updates:**
- Basic usage section to include `require`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,8 +49,9 @@ Then, transform incoming params in your controller using `Decanter#decant`:
 # app/controllers/trips_controller.rb
 
   def create
-    trip_params = TripDecanter.decant(params[:trip])
-    @trip = Trip.new(trip_params)
+    trip_params = params.require(:trip) # or params[:trip] if you are not using Strong Parameters
+    transformed_trip_params = TripDecanter.decant(trip_params)
```
Comment (dpikt) [inline README.md]: `decanted_trip_params` or `parsed_trip_params`? Transformed is a little general (although I realize it's the language used at the top of the readme)

**Comment by chawes13** [comment]
@dpikt Do I need to bump `version.rb`?

**Comment by dianacamacho** [comment]
@chawes13  if you aren't using it already: https://github.com/svenfuchs/gem-release super easy way to bump your gem versions and release to ruby gems

**Comment by chawes13** [comment]
@dpikt Happy with a `minor`?

**Comment by dpikt** [comment]
@chawes13 I'd recommend a patch since there were no code changes.

**Pull request merged**